cypress/support/commands.js

```diff
@@ -47,24 +48,39 @@ Cypress.Commands.add('login', (email, username, password) => {
   });
 });
 
-Cypress.Commands.add('createArticle', (title, description, body) => {
-  cy.getCookie('auth').then((token) => {
-    const authToken = token.value;
+Cypress.Commands.add('createArticle', (user, article) => {
+  cy.request({
+    method: 'POST',
+    url: '/api/users/login',
+    body: {
+      user: {
+        email: user.email,
+        password: user.password
+      }
+    }
+  }).then((loginResponse) => {
+    expect(loginResponse.status).to.eq(200);
+    const authToken = loginResponse.body.user.token;
+    cy.setCookie('auth', authToken);
 
     cy.request({
       method: 'POST',
       url: '/api/articles',
+      headers: {
+        Authorization: 'Token ' + authToken
+      },
       body: {
         article: {
-          title,
-          description,
-          body,
-          tagList: []
+          title: article.title,
+          description: article.description,
+          body: article.body,
+          tagList: article.tagList || []
```

Comment:
The property 'tagList' is used here, but in the `article.cy.js` file, the property is named 'taglist'. Ensure consistency in naming to avoid potential issues.